substratevm/src/com.oracle.svm.truffle/src/com/oracle/svm/truffle/TruffleFeature.java

```diff
@@ -319,7 +319,8 @@ public void afterRegistration(AfterRegistrationAccess a) {
                         "Unsupported TruffleRuntime %s (only SubstrateTruffleRuntime or DefaultTruffleRuntime allowed)",
                         runtime.getClass().getName());
 
-        RuntimeClassInitialization.initializeAtBuildTime("com.oracle.graalvm.locator", "Truffle classes are always initialized at build time");
+        // Truffle classes are always initialized at build time
+        RuntimeClassInitialization.initializeAtBuildTime("com.oracle.graalvm.locator");
```

Comment:
If this is unconditional, it should probably go to the `com.oracle.graalvm.locator`?